GitSourceControl/SettingsView.swift

```diff
@@ -8,8 +8,11 @@
 import SwiftUI
 
 struct SettingsView: View {
+    
+    @State private var doSomething: Bool = false
```

Comment:
Hey this works but we don't use this naming convention. Can you rename this to isPremium?